### PR TITLE
doc: add lookup.json options to README

### PR DESCRIPTION
**Pull request by gibm**
Related Issue: #92 

First draft of change to document what options are available for the lookup table.

Suggestions/criticisms/style nits welcome!

@TheAlphaNerd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,22 @@ Options:
...
+For syntax, see [lookup.json](./lib/lookup.json), the available attributes are:
+
+```
+"replace": true              Download module from github repo in package.json
+"master": true               Use the master branch
+"prefix": "v"                Specify the prefix used in the module version.
+"flaky": true                Ignore failures
+"skip": true                 Completely skip the module
+"repo": "https://github.com/pugjs/jade" - Use a different github repo
+"stripAnsi": true            Sanitize and remove the output ansi data
```
Comment (TheAlphaNerd) [inline README.md]: I would leave out stripAnsi for now. I'll bring it in with the tap reporter PR

**Comment by TheAlphaNerd** [comment]
one small nit otherwise LGTM

**Comment by gibm** [comment]
That makes sense, stripAnsi removed.

**Comment by TheAlphaNerd** [comment]
landed in 78f613412ce7470f15e5df706fd409611137d278

**Pull request closed without merging**